diff --git a/deps/mruby-file-stat/mrbgem.rake b/deps/mruby-file-stat/mrbgem.rake
index ff7d076b..133aee1f 100644
--- a/deps/mruby-file-stat/mrbgem.rake
+++ b/deps/mruby-file-stat/mrbgem.rake
@@ -14,12 +14,16 @@ MRuby::Gem::Specification.new('mruby-file-stat') do |spec|
   file config do
     FileUtils.mkdir_p build_dir, :verbose => true
     Dir.chdir build_dir do
+      if build.kind_of? MRuby::CrossBuild
+        host = "--host=#{build.name}"
+      end
+
       if ENV['OS'] == 'Windows_NT'
         _pp 'on Windows', dir
         FileUtils.touch "#{build_dir}/config.h", :verbose => true
       else
         _pp './configure', dir
-        system env, "#{dir}/configure"
+        system env, "#{dir}/configure #{host}"
       end
     end
   end
